src/views/LimitOrders/components/LimitOrderTable/DetailLimitOrderModal.tsx

```diff
@@ -0,0 +1,252 @@
+        <>
+          <Button variant="primary" mt="24px" as="a" external href={formattedOrder.bscScanUrls.created}>
+            {t('View on BSCScan')}
+          </Button>
+          {!isSubmissionPending && (
+            <Button variant="danger" mt="24px" onClick={onCancelOrder}>
+              {t('Cancel Order')}
+            </Button>
+          )}
+        </>
+      ) : (
+        <Button variant="primary" mt="24px" as="a" external href={formattedOrder.bscScanUrls.created}>
+          {t('View order creation on BSCScan')}
+        </Button>
+      )}
+      {isCancelled && bscScanUrls.cancelled && (
+        <Button variant="primary" mt="24px" as="a" external href={bscScanUrls.cancelled}>
+          {t('View order cancellation on BSCScan')}
+        </Button>
+      )}
+      {isExecuted && bscScanUrls.executed && (
+        <Button variant="primary" mt="24px" as="a" external href={bscScanUrls.executed}>
+          {t('View order execution on BSCScan')}
+        </Button>
+      )}
+    </>
+  )
+  return (
+    <StyledModal
+      title={t('Open Order Details')}
+      headerBackground={theme.colors.gradients.cardHeader}
+      style={{ width: '436px' }}
+      onDismiss={onDismiss}
+    >
+      {attemptingTxn ? (
+        <LoadingContent />
+      ) : txHash ? (
+        <TransactionSubmittedContent chainId={chainId} hash={txHash} onDismiss={onDismiss} />
+      ) : cancellationErrorMessage ? (
+        <TransactionErrorContent onDismiss={onDismiss} message={cancellationErrorMessage} />
+      ) : (
+        orderDetails
+      )}
+    </StyledModal>
+  )
+}
+
+interface LimitTradeInfoCardProps {
+  currentPriceExchangeRateText: string
+  currentPriceExchangeRateTextReversed: string
+  limitPriceExchangeRateText: string
+  limitPriceExchangeRateTextReversed: string
+  isOpen: boolean
+  isExecuted: boolean
+  isCancelled: boolean
+  isSubmissionPending: boolean
+  isCancellationPending: boolean
+}
+
+const LimitTradeInfoCard: React.FC<LimitTradeInfoCardProps> = memo(
+  ({
+    limitPriceExchangeRateText,
+    limitPriceExchangeRateTextReversed,
+    isOpen,
+    isExecuted,
+    isCancelled,
+    isSubmissionPending,
+    isCancellationPending,
+  }) => {
+    const { t } = useTranslation()
+
+    return (
+      <InfoCardWrapper>
+        <Box mb="4px">
+          {isOpen && isSubmissionPending && (
+            <Tag outline scale="sm" p="8px" mb="16px" variant="warning">
+              {t('Pending')}
+            </Tag>
+          )}
+          {isExecuted && (
+            <Tag outline scale="sm" p="8px" mb="16px" variant="success">
+              {t('Filled')}
+            </Tag>
+          )}
+          {isCancellationPending && (
+            <Tag outline scale="sm" p="8px" mb="16px" variant="warning">
+              {t('Cancelling')}
+            </Tag>
+          )}
+          {isCancelled && !isCancellationPending && (
+            <Tag outline scale="sm" p="8px" mb="16px" variant="failure">
+              {t('Cancelled')}
```

Comment:
Currently we have `Canceled` word in translation. Should we use the new one with double `l` or use the existing one ?

Reply:
Let's fix it later. I think in predictions some "canceled" strings might come from the status field and not just used as plain text 🥴